torch/distributed/_tensor/dispatch.py

```diff
@@ -163,9 +164,10 @@ def _operator_dispatch(
 
     if mesh is not None and mesh.get_coordinate() is None:
         # For a non-participating device, we do:
-        # 1. if the return type is scalar, all gather the local result
-        # from participating devices, and reduce on the list of results
-        # with appropriate operators.
+        # 1. if the return type is scalar, set the local result to None.
+        # The local results from all devices will then be all-gathered
```

Comment:
nit: can you indent the comments(this line and the below two line) so that it aligned with other comments?